### DISCUSSION TITLE
Gitpod.io support

### DISCUSSION THREAD

**Discussion by kulturbande** [kimai/kimai#3192] [General]
Hey,

I'm currently playing around with [gitpod.io](https://gitpod.io). It is remote developer platform/IDE and I thinking about on adding all necessary configuration to spin up Kimai2 on-the-fly. My question is only, if it would be interessting to have an easier development setup or if it isn't needed at all?

<img width="1680" alt="Aufnahme 2022-03-12 at 14 23 11@2x" src="https://user-images.githubusercontent.com/68833/158019637-abd0ff68-a3c7-4bd8-abee-737192451dc5.png">

**Comment by kevinpapst** [comment]
I am a fan of local installations, but I am not the target group of such a feature. So I'd say: sounds awesome (as long as it does not cost anything). Having a simple way for a new developer to spin up Kimai and eg. fix a simple bug without the need for a local installation is a neat idea.

As I never used gitpod.io before, I have no idea how it works... but I am happy to test it if you want to share your progress.

**Reply by kulturbande** [reply]
I will try it out. Normally I also prefer a local environment, but if you don't work on a regular basis on this topic it is a nice alternative. I'll keep you posted.
